netsetup/connman.py

```diff
@@ -46,6 +46,23 @@ def __get_wifi_technology_path(self):
                 if properties.get('Type') == 'wifi':
                         return obj_path
 
+    def __get_ethernet_service_properties(self):
+        if not self.manager:
+            logging.error('%s: unable to get ethernet technology.')
+            return
+
+        for obj_path, properties in self.manager.GetServices():
+                if properties.get('Type') == 'ethernet':
+                        return properties
+
+    def __parse_mac_address(self, mac):
+        return mac.replace(':', '_')
+
+    def __get_mac_address(self):
+        properties_ethernet = self.__get_ethernet_service_properties()
+        mac = properties_ethernet.get('Ethernet').get('Address')
```

Comment:
There should be a verification if there is 'Ethernet'technology. Because in case of the raspberry pi 0 there is no such interface.